src/serverenvironment.cpp

```diff
@@ -290,16 +290,17 @@ void LBMManager::applyLBMs(ServerEnvironment *env, MapBlock *block, u32 stamp)
 void fillRadiusBlock(v3s16 p0, s16 r, std::set<v3s16> &list)
 {
 	v3s16 p;
+	s32 r2 = r * r;
 	for(p.X=p0.X-r; p.X<=p0.X+r; p.X++)
-		for(p.Y=p0.Y-r; p.Y<=p0.Y+r; p.Y++)
-			for(p.Z=p0.Z-r; p.Z<=p0.Z+r; p.Z++)
-			{
-				// limit to a sphere
-				if (p.getDistanceFrom(p0) <= r) {
-					// Set in list
-					list.insert(p);
-				}
-			}
+	for(p.Y=p0.Y-r; p.Y<=p0.Y+r; p.Y++)
+	for(p.Z=p0.Z-r; p.Z<=p0.Z+r; p.Z++)
```

Comment:
Needs code style fixes (lines were touched).

Reply:
Yep.

Actually... I'm going to take back this change anyway for two reasons: **(1)** This did not show in the profiler - I just fixed it ...  because and **(2)** I'm worried about overflow. getDistanceFromSQ is declared as `T getDistanceFromSQ (const vector3d< T > &other)` so for v3s16 I'd be worried about overflow.